server-spi-private/src/main/java/org/keycloak/exportimport/ImportProviderFactory.java

```diff
@@ -17,10 +17,18 @@
 
 package org.keycloak.exportimport;
 
+import java.util.Map;
+
+import org.keycloak.models.KeycloakSession;
 import org.keycloak.provider.ProviderFactory;
 
 /**
  * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
  */
 public interface ImportProviderFactory extends ProviderFactory<ImportProvider> {
+
+    default ImportProvider create(KeycloakSession session, Map<String, String> overrides) {
+        return create(session);
+    }
```

Comment:
Should we rather have the default impl for `create(KeycloakSession)` to avoid mistakenly ignoring the overrides? If it's part of the interface, IMHO the most specific method should require implementation, with the less specific one having a default.

However, it'll require changes to other import providers, so if we want to keep the changes here less involved, let's keep as is.

Reply:
> However, it'll require changes to other import providers, so if we want to keep the changes here less involved, let's keep as is.

Yes, the main goal is to not be a breaking change. I realize that even using a default method is not guarenteed to be non-breaking as someone could already have create(KeycloakSession, Map<other bounds>) method - that was part of the reasoning of using a separate interface. But this is not likely and easy to fix, so it seems fine to keep it this way.

Reply:
Import Provider is not a public SPI, we can do breaking changes as we like.